cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -560,6 +563,7 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 		exportPaths = append(exportPaths, path.Join(jobTarget.OsbuildArtifact.ExportName, jobTarget.OsbuildArtifact.ExportFilename))
 	}
 
+	logWithId.Infof("Extra env: %s", strings.Join(extraEnv, " "))
```

Comment:
Maybe just `logWithId.Infof("Extra env: %q", extraEnv)` as this will give us automatic quoting if things like spaces are part of the env?

Reply:
It will not: https://go.dev/play/p/2gehrpitrhh

Reply:
Hey @lzap - thanks for the golang play link - I'm slightly confused though, the link uses `%s` but I suggested `%q` which will add quote AFAICT (cf. https://go.dev/play/p/4tZBweD8jFC) - or am I misunderstanding/missing something maybe?

Reply:
Oh I have missed that difference completely, sure, this is indeed more readable. Amended the change.